src/TokensTracker/TokensTracker/config.json

```diff
@@ -0,0 +1,11 @@
+{
+	"PluginConfiguration": {
```

Comment:
Add an option to decide which standards to be tracked?

Reply:
What do you think? @superboyiii

Reply:
We already combined Nep11 tracker and Nep 17 tracker into one plugin, I think default tracker all is OK.